packages/app/src/cli/services/dev/ui/components/Dev.tsx

```diff
@@ -187,9 +187,11 @@ const Dev: FunctionComponent<DevProps> = ({abortController, processes, previewUr
                   {devPreviewEnabled ? <Text color="green">✔ on</Text> : <Text color="red">✖ off</Text>}
                 </Text>
               ) : null}
-              <Text>
-                {figures.pointerSmall} Press <Text bold>g</Text> {figures.lineVertical} open the GraphiQL Explorer in your browser
-              </Text>
+              {graphiqlUrl ? (
+                <Text>
+                  {figures.pointerSmall} Press <Text bold>g</Text> {figures.lineVertical} open the GraphiQL Explorer in your browser
```

Comment:
I know I'm late to the party and I think you already talked to Meredith, but have you considered adding this to the dev console instead of showing so many options and shortcuts in the terminal?